Makefile

```diff
@@ -142,7 +142,18 @@ go-mod-cache: go.sum
 	@echo "--> Download go modules to local cache"
 	@go mod download
 
-.PHONY: all build install clean wasmvmlib build-reproducible
+get-heighliner:
+	git clone https://github.com/strangelove-ventures/heighliner.git $(TEMPDIR)/heighliner
```

Comment:
https://github.com/strangelove-ventures/heighliner/releases/tag/v1.5.3
How about downloading binary?

Reply:
I set [v1.5.3](https://github.com/strangelove-ventures/heighliner/releases/tag/v1.5.3) for heighliner